rocks-db/src/dump_client.rs

```diff
@@ -275,6 +296,7 @@ impl Storage {
             if rx_indexes_cloned.is_empty() {
                 continue;
             } else if let Ok(indexes) = rx_indexes_cloned.try_recv() {
+                let start = Instant::now();
                 for (key, index) in indexes {
                     let metadata_url = index
                         .metadata_url
```

Comment:
I'd advise on increasing some metric on every .send to any channel (like the tx_metadata_cloned)